sdk/python/setup.py

```diff
@@ -61,6 +61,7 @@
     "tabulate==0.8.*",
     "toml==0.10.*",
     "tqdm==4.*",
+    "boto3==1.17.*",
```

Comment:
If this dependency is aws specific then it needs to be in its own `extras` object so that it isnt installed by default